spec/models/book_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+describe Book do
+  subject(:book) { create(:book) }
+
+  context 'Create a book with Faker' do
```

Comment:
We usually don't declare a context to specify we are testing the creation of a model with faker. A common pattern to follow is
```ruby
describe Book do
  subject(:book) { build(:book) }

  it { is_expect.to be_valid }

  # Specs related with validations and relations (shoulda_matchers)
  ...

  # specs of model's methods
  ...
end
```

A few things about that snippet:
- I'm not creating a book, I'm building it! A `create()` performs a `.new` and a `.save`, so the record is persisted in the DB. A `build()` just does a `.new` and returns the instance.
- The spec to test if the instance created by factory_bot is valid has no name and makes an expectation on `is_expected`. It's a little tricky to understand, but is the same of `it { expect(book).to be_valid }`. [Here](https://relishapp.com/rspec/rspec-core/docs/subject/one-liner-syntax) is the documentation!
- `be_valid` is just a shortcut of sending the message #valid? to the instance and expect a `true` as response. So, you could rewrite that spec as `it { expect(book.valid?).to be(true) }`